<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ladder</class>
 <widget class="QDialog" name="Ladder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Статистика: количество ступенек</string>
  </property>
  <widget class="QCustomPlot" name="DDAPlot" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="BresenhemPlot" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="BresenhemIntegerPlot" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="BresenhemAntiPlot" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
